sdk/js/src/cosmwasm/address.ts

```diff
@@ -1,9 +1,43 @@
+import { bech32 } from "bech32";
 import { keccak256 } from "ethers/lib/utils";
 import { isNativeDenom } from "../terra";
+import {
+  CHAIN_ID_INJECTIVE,
+  CHAIN_ID_TERRA2,
+  CosmWasmChainId,
+  isTerraChain,
+} from "../utils";
 
-export function buildTokenId(address: string) {
+export function canonicalAddress(humanAddress: string) {
+  return new Uint8Array(bech32.fromWords(bech32.decode(humanAddress).words));
+}
+export function humanAddress(
```

Comment:
ditto `canonicalAddress`

Reply:
Also, removed humanAddress from this file.